python/pdo/contract/contract.py

```diff
@@ -161,7 +163,7 @@ def create_update_request(self, request_originator_keys, expression, enclave_ser
             request_originator_keys,
             self,
             enclave_service=enclave_service,
-            expression = expression)
+            invocation_request = expression)
 
```

Comment:
this change seems misleading, given that a request till now meant evaluate "expression on a specific state". Especially, with the new term invocation_response, which is likely interpreted as invocation_response = function(invocation_request) = function(expression, state)

Reply:
i don't understand this comment. or why it is relevant?

Reply:
was just referring to the change in the variable name from expression to invocation_request. i guess i am being a bit too formal on the choice of names. invocation_request is OK.